src/index.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash'; // eslint-disable-line
```

Comment:
it seems that there is no  need to  use this library and it seems useless
kindly remove this library